### PR TITLE
add quantize dependency to tooltips of sync_enable and beatsync

### PR DESCRIPTION
**Pull request by daschuer**
This is a follow up PR to https://github.com/mixxxdj/mixxx/pull/1679 for not breaking string freeze in the 2.1 branch

### CODE REVIEW

File: src/controllers/controlpickermenu.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     QMenu* syncMenu = addSubmenu(tr("Sync"));
     addDeckAndSamplerControl("sync_enabled", tr("Sync Mode"),
-                             tr("Tap to sync, hold to enable sync mode"), syncMenu);
+                             tr("Tap to sync (tempo and phase depending on quantize), hold to enable permanent sync"), syncMenu);
```
Comment (Be-ing) [inline src/controllers/controlpickermenu.cpp]: "Tap to sync tempo (and phase with quantize enabled)" is a bit clearer

File: src/controllers/controlpickermenu.cpp [hunk 2]
```diff
@@ -114,7 +114,7 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
     addDeckAndSamplerControl("sync_mode", tr("Sync Mode"),
                              tr("Sync mode 3-state toggle (OFF, FOLLOWER, MASTER)"), syncMenu);
     addDeckAndSamplerControl("beatsync", tr("Beat Sync One-Shot"),
-                             tr("One-time beat sync (tempo and phase)"), syncMenu);
+                             tr("One-time beat sync (tempo and phase depending on quantize)"), syncMenu);
```
Comment (Be-ing) [inline src/controllers/controlpickermenu.cpp]: "One-time beat sync tempo (and phase with quantize enabled)"

**Review by Be-ing — CHANGES_REQUESTED** [review]

**Comment by daschuer** [comment]
Done.

**Comment by daschuer** [comment]
Merge?

**Comment by Be-ing** [comment]
Should this go to 2.2 or master?

**Comment by daschuer** [comment]
To master, because it needs to be translated.

**Comment by Be-ing** [comment]
Okay

**Pull request merged**